admin/workspace-providers/deployment.md

```diff
@@ -63,119 +57,80 @@ kubectl config set-context --current --namespace=coder
-      --install \
-      --set envproxy.token=[REMOTE_ENVPROXY_TOKEN] \
-      --set envproxy.accessURL=[HOSTNAME] \
-      --set ingress.host=[HOSTNAME_WITH_NO_PROTOCOL] \
-      --set envproxy.clusterAddress=[CLUSTER_ADDRESS] \
-      --set cemanager.accessURL=[CEMANAGER_ACCESS_URL]
+1. Create a service account in the namespace that you specified in the previous
+   step (Coder will use this account to provision workspaces):
+
+   ```console
+   kubectl apply -n default -f - <<EOF
+   apiVersion: v1
+   kind: ServiceAccount
+   metadata:
+     name: coder
+   ---
+   apiVersion: rbac.authorization.k8s.io/v1
+   kind: Role
+   metadata:
+     name: coder
+   rules:
+     - apiGroups: ["", "apps", "networking.k8s.io"] # "" indicates the core API group
+       resources: ["persistentvolumeclaims", "pods", "deployments", "services", "secrets", "pods/exec","pods/log", "events", "networkpolicies"]
+       verbs: ["create", "get", "list", "watch", "update", "patch", "delete", "deletecollection"]
+     - apiGroups: ["metrics.k8s.io", "storage.k8s.io"]
+       resources: ["pods", "storageclasses"]
+       verbs: ["get", "list", "watch"]
+   ---
+   apiVersion: rbac.authorization.k8s.io/v1
+   kind: RoleBinding
+   metadata:
+     name: coder
+   subjects:
+     - kind: ServiceAccount
+       name: coder
+   roleRef:
+     kind: Role
+     name: coder
+     apiGroup: rbac.authorization.k8s.io
+   EOF
    ```
 
-   Optionally, you can provide additional helm configuration values by providing
-   a `values.yaml` file and adding the argument `-f my-values.yaml` to the
-   generated command. Helm values control attributes of the workspace provider,
-   including dev URLs, Kubernetes storage classes, SSH, and more. See the
-   [Workspace provider Helm chart values]("https://github.com/cdr/enterprise-helm/blob/workspace-providers-envproxy-only/README.md")
-   for more details.
+   You should get a response similar to the following:
 
-   For installations using HTTPS, you must
-   [ensure the deployment has a valid certificate](../../guides/ssl-certificates/index.md).
-
-   If you're unfamiliar with the helm configuration values file, see our doc on
-   [updating a helm chart](../../guides/admin/helm-charts.md)
-
-1. Once the helm chart is successfully deployed, fetch the ingress address:
-
-   ```bash
-   kubectl get ingress web-ingress
+   ```console
+   serviceaccount/coder created
+   role.rbac.authorization.k8s.io/coder created
+   rolebinding.rbac.authorization.k8s.io/coder created
    ```
 
-   Use this IP to create a DNS record for the provided hostname of the workspace
-   provider.
-
-1. Once the Helm chart has deployed successfully, you should see the workspace
-   provider in a `ready` state on the **Workspace provider admin** page.
-
-   ![Workspace providers admin](../../assets/admin/workspace-providers-admin.png)
-
-1. From the **Workspace provider admin** page, add the desired organizations to
-   its allowlist.
+1. Get the tokens required:
```

Comment:
@f0ssel can you explain a bit more what these tokens are for/what they do?

Reply:
added some edits in 051006b and 0f4e3dc